process/css-styleguide/heading/heading.css

```diff
@@ -0,0 +1,179 @@
+@import '../default.css';
+
+.typography {
+  font-family: 'Poppins', sans-serif;
+  color: var(--valhalla-500);
+}

```

Comment:
We have 2 font-family in the design system: `poppins` and `inter`. I suggest you add classes for these 2 fonts.